test/integration/isolation_test.exs

```diff
@@ -6,18 +6,18 @@ defmodule Oban.Integration.IsolationTest do
   @moduletag :integration
 
   test "multiple supervisors can be run simultaneously" do
-    start_supervised_oban!(name: ObanA, queues: [alpha: 1], plugins: [Oban.Plugins.Pruner])
-    start_supervised_oban!(name: ObanB, queues: [gamma: 1], plugins: [Oban.Plugins.Pruner])
+    name1 = start_supervised_oban!(queues: [alpha: 1], plugins: [Oban.Plugins.Pruner]).name
+    start_supervised_oban!(queues: [gamma: 1], plugins: [Oban.Plugins.Pruner])
 
-    insert!(ObanA, %{ref: 1, action: "OK"}, [])
+    insert!(name1, %{ref: 1, action: "OK"}, [])
 
     assert_receive {:ok, 1}
```

Comment:
This test loses a bit of its pizazz now that _all_ of the names are dynamic. I don't believe we need to insert a job to prove that it works. Simply starting up the separate supervision trees proves that there aren't conflicts.

```elixir
name1 = start_supervised_oban!(queues: [gamma: 1], plugins: [Oban.Plugins.Pruner])
name2 = start_supervised_oban!(queues: [gamma: 1], plugins: [Oban.Plugins.Pruner])

assert name1 != name2
```